### PR TITLE
Added progress bar to nav bar

### PR DESCRIPTION
**Pull request by llau6**
- Removed debug logs

image
- Added progress bar
- Added current location

image
- Added icon (pen) for accessing Scratch
- Changed social media icon to a globe

### CODE REVIEW

File: runestone/activecode/test/_templates/plugin_layouts/sphinx_bootstrap/static/bootstrap-sphinx.js [hunk 1]
```diff
@@ -52,6 +54,7 @@
   };
 
   $(document).ready(function () {
+    console.log("bootstrap-sphinx.js running")
```
Comment (bnmnetp) [inline runestone/activecode/test/_templates/plugin_layouts/sphinx_bootstrap/static/bootstrap-sphinx.js]: This looks like debug code that should be removed before I accept the PR

File: runestone/activecode/test/test_activecode.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 from runestone.unittest_base import module_fixture_maker, RunestoneTestCase
 
 setUpModule, tearDownModule = module_fixture_maker(__file__)
-
+print "test_activecode.py"
```
Comment (bnmnetp) [inline runestone/activecode/test/test_activecode.py]: The runestone components are Python3 compatible, so print should use parenthesis.  Is this also debug code that should be removed?

File: runestone/common/js/navhelp.js [hunk 3]
```diff
@@ -3,7 +3,7 @@
  */
 
 /* Sets up the interactive navhelp */
-
+var navHelp = "this is from navHelp.js"
```
Comment (bnmnetp) [inline runestone/common/js/navhelp.js]: Does this serve a purpose?

File: runestone/common/project_template/_templates/plugin_layouts/sphinx_bootstrap/globaltoc.html [hunk 4]
```diff
@@ -1,3 +1,6 @@
+<script>
+  console.log("globaltoc");
+</script>
```
Comment (bnmnetp) [inline runestone/common/project_template/_templates/plugin_layouts/sphinx_bootstrap/globaltoc.html]: More debug?

**Comment by bnmnetp** [comment]
@llau6 @ericsonga I like the progress in chapter, progress in book on the right side.  But the left side of chapter / subchapter location has replaced the navigation dropdown that allows you to quickly move between chapters.  

I would like to keep that dropdown, or at least keep it as a build option if you prefer one or the other.

**Comment by llau6** [comment]
@bnmnetp It looks like on the book I was working on (apcsareview) there doesn't exist a Change Dropdown. I went ahead and downloaded thinkcspy and did see the dropdown you mentioned. Do you know why the dropdown wouldn't appear on apcsareview? I figured the answer would be in conf.py under `html_theme_options` as an option. I see that apcsareview has `'navbar_site_nam e': "Chapters"` listed as an option, but there still is no Chapter Dropdown. Do you know if this is how to turn on and off the Chapter Dropdown?

**Comment by llau6** [comment]
@ericsonga @bnmnetp I have removed the debug logs and added the progress bar logic inside its own filed called `progress.html`. Please let me know if you run into any issues that I need to fix.

File: runestone/activecode/js/activecode.js [hunk 5]
```diff
@@ -61,8 +61,8 @@ ActiveCode.prototype.init = function(opts) {
 
     this.history = [this.code];
     this.createEditor();
-    this.createOutput();
-    this.createControls();
+    // this.createOutput();
```
Comment (bnmnetp) [inline runestone/activecode/js/activecode.js]: Ugh!  Why were these commented out!??

**Pull request merged**